background.js

```diff
@@ -34,6 +34,13 @@ chrome.commands.onCommand.addListener(function (command) {
         update_url_regex = RegExp("\\?u\=[0-9]");
         update_acc = "?u=" + account_num
       }
+      
+      // Google Classroom
+      const classroom_regex = "https:\/\/classroom.google.com\/u\/[0-9].*";
+      if (current_url.match(classroom_regex)) {
+        update_url_regex = RegExp("\\?u\=[0-9]");
```

Comment:
this is incorrect
Check google classroom URL format
https://classroom.google.com/u/2/h
And test PR before sending